### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by blampe** [pulumi/pulumi-go-helmbase#10]
This repository is currently untested. Which is bad.

**Comment by rshade** [comment]
Is there a way we can just make it trigger updates & tests on cert-manager, and nginx?

**Comment by blampe** [comment]
> Is there a way we can just make it trigger updates & tests on cert-manager, and nginx?

Yeah, we could write a smoke test that essentially just invokes Construct on one of those charts to sanity check things.
